### PR TITLE
Allow unbiased simulation to collect CVs

### PR DESCRIPTION
**Pull request by InnocentBug**
Not the most elegant implementation since we are still adding zeros in the simulations.
But since should only be test simulations, I think it is good to go.

### CODE REVIEW

**Comment by InnocentBug** [comment]
Maybe down the line we should improve this implementation.

I can see people using PySAGES just for this.
Implementing their own CV to measure an observable in their simulation and log it.

It's easy to do and depending on your backend (Hoomd easy on cpu, OpenMM Idk: hard) it is worthwhile.
And if ppl use it like this, we should circumvent the addition of the zero force altogether.

**Comment by pabloferz** [comment]
Can we call it `Unbiased`?

Regarding avoiding adding the forces, I think we can do it by setting the bias to `None`, and changing the backends code to conditionally add them if `state.bias is not None`.

**Comment by InnocentBug** [comment]
Great suggestions, I implemented both.

File: examples/hoomd-blue/unbiased/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+start.gsd
```
Comment (pabloferz) [inline examples/hoomd-blue/unbiased/.gitignore:1]: Isn't this covered by our root `.gitignore`?

Comment (InnocentBug) [reply examples/hoomd-blue/unbiased/.gitignore:1]: I think it is.
I copied it from harmonic_bias example.
Does it hurt?

Comment (pabloferz) [reply examples/hoomd-blue/unbiased/.gitignore:1]: It doesn't, it's just my picky side speaking. I can ignore it for now.

**Comment by pabloferz** [comment]
I added a commit to avoid computing the Jacobians as well.

**Review by pabloferz — APPROVED** [review]

**Pull request merged**